### PR TITLE
Snyder la107.an image r c sys fd y.r

### PR DESCRIPTION
**Pull request by abigailsnyder**

### CODE REVIEW

File: R/zchunk_LA107.an_IMAGE_R_C_Sys_Fd_Y.R [hunk 1]
```diff
@@ -36,61 +38,251 @@ module_aglu_LA107.an_IMAGE_R_C_Sys_Fd_Y_DISABLED <- function(command, ...) {
...
+
+    # Combine the mixed production table, L107.an_Prod_Mt_ctry_C_mix_Y, and the pastoral production table, L107.an_Prod_Mt_ctry_C_past_Y,
+    # to form a table organizing the amount of animal production by country, commodity, system, and year:
+    L107.an_Prod_Mt_ctry_C_Sys_Y <- bind_rows( L107.an_Prod_Mt_ctry_C_mix_Y, L107.an_Prod_Mt_ctry_C_past_Y )
+
+
+
+
+
+
```
Comment (bpbond) [inline R/zchunk_LA107.an_IMAGE_R_C_Sys_Fd_Y.R]: Tighten up these blank spaces a bit please

File: R/zchunk_LA107.an_IMAGE_R_C_Sys_Fd_Y.R [hunk 2]
```diff
@@ -36,61 +38,251 @@ module_aglu_LA107.an_IMAGE_R_C_Sys_Fd_Y_DISABLED <- function(command, ...) {
...
+      add_comments("animal production data.") %>%
       add_legacy_name("L107.an_Prod_Mt_R_C_Sys_Fd_Y") %>%
-      add_precursors("precursor1", "precursor2", "etc") %>%
+      add_precursors("common/iso_GCAM_regID",
+                     "temp-data-inject/L100.IMAGE_an_Prodmixfrac_ctry_C_Y",
+                     "temp-data-inject/L100.IMAGE_an_Feedfrac_ctry_C_Sys_Fd_Y",
+                     "temp-data-inject/L100.IMAGE_an_FeedIO_ctry_C_Sys_Y",
+                     "temp-data-inject/L105.an_Prod_Mt_ctry_C_Y") %>%
       # typical flags, but there are others--see `constants.R`
       add_flags(FLAG_NO_TEST, FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
```
Comment (bpbond) [inline R/zchunk_LA107.an_IMAGE_R_C_Sys_Fd_Y.R]: Uh oh--it looks like this FLAG_NO_TEST was left in, which means the chunk's outputs weren't being tested. Can you double-check please?

Note to self: this is probably a stupid thing to have the skeleton script drop in.

File: R/zchunk_LA107.an_IMAGE_R_C_Sys_Fd_Y.R [hunk 3]
```diff
@@ -36,61 +38,251 @@ module_aglu_LA107.an_IMAGE_R_C_Sys_Fd_Y_DISABLED <- function(command, ...) {
...
     # There's also a `same_precursors_as(x)` you can use
     # If no precursors (very rare) don't call `add_precursor` at all
-    tibble() %>%
-      add_title("descriptive title of data") %>%
-      add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
+    L107.an_Prod_Mt_R_C_Sys_Fd_Y %>%
+      add_title("Animal production by GCAM region / commodity / system / feed type / year") %>%
+      add_units("Million tonnes (Mt)") %>%
```
Comment (bpbond) [inline R/zchunk_LA107.an_IMAGE_R_C_Sys_Fd_Y.R]: Here and below, should be "Megatons (Mt)"

File: R/zchunk_LA107.an_IMAGE_R_C_Sys_Fd_Y.R (hunk 2 above)
Comment (abigailsnyder) [inline R/zchunk_LA107.an_IMAGE_R_C_Sys_Fd_Y.R]: whoopsie doodle, I'll get on that. That's great to know for the future!

Comment (abigailsnyder) [inline R/zchunk_LA107.an_IMAGE_R_C_Sys_Fd_Y.R]: It's failing but the error for each output is
1. Failure: matches old data system output (@test_oldnew.R#54) -----------------
length(oldf) == 1 isn't true.
Either zero, or multiple, comparison datasets found for L107.an_Feed_Mt_R_C_Sys_Fd_Y.csv


When I search, each .csv file is present in both gcamdata/outputs and gcamdata/tests/testthat/outputs
,

Comment (bpbond) [inline R/zchunk_LA107.an_IMAGE_R_C_Sys_Fd_Y.R]: Hang on, let me look.

Comment (bpbond) [inline R/zchunk_LA107.an_IMAGE_R_C_Sys_Fd_Y.R]: OK, I'm pushing an update to this PR with files and tweaks to your chunk...

Comment (abigailsnyder) [inline R/zchunk_LA107.an_IMAGE_R_C_Sys_Fd_Y.R]: Tests are passing on this, thank you!

**Comment by bpbond** [comment]
Last request - @abigailsnyder it looks like `tests/testthat/Rplots.pdf` got added by accident - can you remove please?

**Pull request merged**